.github/workflows/e2etests.yaml

```diff
@@ -26,15 +26,21 @@ jobs:
         run: |
             if [ ${{ github.event_name }} == "release" ];then
                 echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
-            else 
-                echo "version=edge" >> $GITHUB_ENV
-            fi      
-      - name: Get latest release tag
-        id: gettag
+            else
+                echo "version=$(steps.getadapter.outputs.release)" >> $GITHUB_ENV
+            fi
+      - name: Get latest stable release of adapter
+        id: getadapter
         uses: pozetroninc/github-action-get-latest-release@master
         with:
           repository: linkerd/linkerd2
-          excludes: prerelease, draft, edge 
+          excludes: prerelease, draft, edge     
```

Comment:
I wonder if this Github action has any cache for the latest release. If yes, please add those parameters to enable the cache.

Reply:
It have cache for latest release but the issue talks about latest and stable version couldn't find any cache for that or any github action

Reply:
@duckling69 Thanks for clarify it